src/webui/api/torrentscontroller.cpp

```diff
@@ -2143,3 +2143,20 @@ void TorrentsController::onMetadataDownloaded(const BitTorrent::TorrentInfo &inf
             iter.value().setTorrentInfo(info);
     }
 }
+
+void TorrentsController::setCommentAction()
+{
+    requireParams({u"hash"_s, u"comment"_s});
```

Comment:
Wouldn't it make sense to allow setting the same comment to multiple torrent at once?